### ISSUE TITLE
Loss curves

### ISSUE THREAD

**Issue by david-az** [open-mmlab/mmselfsup#101]
Hi, thanks everyone for the great work !
I am trying to train BYOL on a custom dataset but struggling to get good results. Loss during training is very noisy (while still going down quite a bit), and I can't find anywhere what BYOL loss curves should look like.

Would it be possible to add the loss curves for the different benchmarks in order to compare with custom trainings ?

Thanks !

**Comment by Jiahao000** [comment]
You may first try to reproduce the results on ImageNet. When training on a custom dataset, you may need to change the default hyper-parameters.
